owned-manifests/machine-api-controllers.yaml

```diff
@@ -31,7 +31,7 @@ spec:
       - key: CriticalAddonsOnly
         operator: Exists
       - effect: NoExecute
-        key: node.alpha.kubernetes.io/notReady
+        key: node.kubernetes.io/not-ready
         operator: Exists
```

Comment:
@ravisantoshgudimetla why aren't we setting `effect` and `tolerationSeconds` explicitly here but we do above?